src/content/docs/guides/platform-management/configure-workspaces.mdx

```diff
@@ -14,13 +14,80 @@ You can configure workspaces in two ways:
 
 ## Manage workspaces dynamically
 
-Our [platform command-line utility](/reference/platform/command-line-interface)
-supports managing at runtime.
+The Tenzir Platform CLI allows administrators to create, modify, and delete
+workspaces on demand.
 
-:::caution[WIP]
-Extract content from CLI docs to here.
+:::caution[On-premise setup required]
+This CLI functionality requires an on-premise platform deployment, available
+with the [Sovereign Edition](https://tenzir.com/pricing).
 :::
 
+Only local platform administrators can manage workspaces dynamically. The
+[`TENZIR_PLATFORM_OIDC_ADMIN_RULES` variable](/guides/platform-setup/configure-identity-provider)
+defines who's an administrator in your platform deployment.
+
+### Creating and managing workspaces
+
+You can create workspaces for either individual users or organizations. When
+you create a user workspace, it's automatically configured with access for that
+specific user. Organization workspaces start with no access rules, giving you
+full control over who can access them.
+
+For a detailed overview, of the available commands, take a look at our
+[CLI reference documentation](/reference/platform/command-line-interface).
+
+### Example: Setting up an organization workspace
+
+Let's walk through creating a workspace for the fictional "Scrooge & Marley Counting House" organization:
+
+1. **Create the organization workspace:**
+
+   ```bash
+   tenzir-platform admin create-workspace organization scrooge-marley --name "Scrooge & Marley Counting House"
+   ```
+
+   This creates a workspace with the organization ID `scrooge-marley` and the display name "Scrooge & Marley Counting House".
+
+2. **Configure access for employees with company email addresses:**
+
+   ```bash
+   tenzir-platform admin add-auth-rule email-domain <workspace_id> <connection> '@scroogemarley.com'
+   ```
+
+3. **Add a specific user:**
+
+   The specific format of the user id depends on the OIDC provider you configured.
+   The provided id must match the `sub` of the generated OIDC token in order
+   to allow access to the workspace.
+
+   ```bash
+   tenzir-platform admin add-auth-rule user <workspace_id> 'sub|12345678901'
+   ```
+
+4. **Grant access to users with the "accountant" role:**
+
+   ```bash
+   tenzir-platform admin add-auth-rule organization-role <workspace_id> <connection> roles accountant organization scrooge-marley
+   ```
+
+If you later need to remove the workspace:
```

Comment:
This comes a bit abrupt. Should we rather have a separate section for removing a workspace?